frontend/public/locales/en/apply.json

```
@@ -93,6 +93,31 @@
       "sun-life": "https://www.sunlife.ca/en/"
     }
   },
+  "tax-filing": {
+    "page-title": "Tax filing",
+    "form-instructions": "Have you and your spouse or common-law partner (if applicable) filed your tax return for 2023 and received your Notice of Assessment?",
+    "radio-options": {
+      "yes": "Yes",
+      "no": "No"
+    },
+    "error-message": {
+      "tax-filing-required": "Select whether or not you have filed your taxes"
+    },
+    "back-btn": "Back",
+    "continue-btn": "Continue"
+  },
+  "file-your-taxes": {
+    "page-title": "File your taxes",
+    "ineligible-to-apply": "You are not eligible to apply for the Canadian Dental Care Plan at this time.",
+    "tax-not-filed": "You, your spouse or common-law partner have not filed a tax return for 2023 with the Canada Revenue Agency.",
+    "unable-to-assess": "We use the information from your tax file to verify that the income and residency eligibility requirements are met for the plan.",
+    "tax-info": "For more information on how to file your taxes, visit <taxInfo>Get ready to do your taxes</taxInfo>.",
+    "apply-after": "You can apply for the plan after you or your spouse or common-law partner have filed your taxes and received your Notice of Assessment.",
```

From Figma

```suggestion
    "tax-not-filed": "You, your spouse or common-law partner have not received your Notice of Assessment for {{taxYear}} from the Canada Revenue Agency.",
    "unable-to-assess": "We use the information from your tax file to verify that the income and residency eligibility requirements for the Canadian Dental Care Plan are met.",
    "tax-info": "For more information on how to file your taxes, visit <taxInfo>Get ready to do your taxes</taxInfo>.",
    "apply-after": "You can apply for the plan only after you or your spouse or common-law partner have filed your taxes and received your Notice of Assessment.",
```

(Same comment applies for French)